lib/fast_gettext/translation.rb

```diff
@@ -20,13 +20,15 @@ def self.included(klas)  #:nodoc:
     def _(key, &block)
       FastGettext.cached_find(key) or (block ? block.call : key)
     end
+    alias :gettext :_
 
     #translate pluralized
     # some languages have up to 4 plural forms...
     # n_(singular, plural, plural form 2, ..., count)
     # n_('apple','apples',3)
     def n_(*keys, &block)
       count = keys.pop
+      keys = keys.flatten if keys.count == 1 && keys.respond_to?(:flatten)
```

Comment:
can we not add that ... I want users to call methods with correct arguments and not add extra overhead for everyone who does it right ...

Reply:
I'm ok leaving out. I was just trying to make fast_gettext a complete drop-in replacement for gettext.